docker-compose.yml

```diff
@@ -1,26 +1,134 @@
-    restart: on-failure
+    image: rabbitmq:3
+    networks:
+      - net_rabbit_bridge_request
+      - net_rabbit_bridge_collected
+      - net_rabbit_bridge_affirmation
+      - net_rabbit_bridge_senderhome
+      - net_rabbit_bridge_senderforeign
+    restart: unless-stopped
+    volumes: ['~/bridge_data/rabbitmq:/var/lib/rabbitmq/mnesia']
+  redis:
+    cpus: 0.1
+    mem_limit: 500m
+    command: [redis-server, --appendonly, 'yes']
+    hostname: redis
+    image: redis:4
+    networks:    
+      - net_db_bridge_request
+      - net_db_bridge_collected
+      - net_db_bridge_affirmation
+      - net_db_bridge_senderhome
+      - net_db_bridge_senderforeign
+    restart: unless-stopped
+    volumes: ['~/bridge_data/redis:/data']
   bridge:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+    restart: on-failure
+  bridge_request:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
+      - VALIDATOR_ADDRESS_PRIVATE_KEY=${VALIDATOR_ADDRESS_PRIVATE_KEY}
+    restart: unless-stopped
+    entrypoint: npm run watcher:signature-request
+    networks:
+      - net_db_bridge_request
+      - net_rabbit_bridge_request
+  bridge_collected:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
+    restart: unless-stopped
+    entrypoint: npm run watcher:collected-signatures
+    networks:
+      - net_db_bridge_collected
+      - net_rabbit_bridge_collected
+  bridge_affirmation:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
+    restart: unless-stopped
+    entrypoint: npm run watcher:affirmation-request
+    networks:
+      - net_db_bridge_affirmation
+      - net_rabbit_bridge_affirmation
+  bridge_senderhome:
+    cpus: 0.1
+    mem_limit: 500m
+    build: .
+    command: 'true'
+    env_file: ./.env
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
+      - VALIDATOR_ADDRESS_PRIVATE_KEY=${VALIDATOR_ADDRESS_PRIVATE_KEY}
+    restart: unless-stopped
+    entrypoint: npm run sender:home
+    networks:
+      - net_db_bridge_senderhome
+      - net_rabbit_bridge_senderhome
+  bridge_senderforeign:
+    cpus: 0.1
+    mem_limit: 500m
     build: .
+    command: 'true'
     env_file: ./.env
-    environment:
-      - NODE_ENV=production
-      - QUEUE_URL=amqp://rabbit
-      - REDIS_URL=redis://redis
-    command: "true"
-    restart: on-failure
\ No newline at end of file
+    environment: 
+      - NODE_ENV=production 
+      - VALIDATOR_ADDRESS=${VALIDATOR_ADDRESS}
```

Comment:
I think we don't need `VALIDATOR_ADDRESS` here since the address will be generated automatically based on the private key provided in the next line.